src/content/basics/nodepools/index.md

```diff
@@ -24,7 +24,7 @@ availability zone distribution, even if some workloads wouldn't require the incr
 Node pools are independent groups of worker nodes belonging to a cluster, where all nodes within a pool share a
 common configuration. You can combine any type of node pool within one cluster. Node pools can differ regarding:
 
-- EC2 instance type
+- Instance type
```

Comment:
It's hard to explain things when cloud providers use different names for the same concept. I was thinking if it would be better to write `Instance type / VM size`?

Reply:
Would Machine type encapsulate both ?

Reply:
+1 if you agree with `machine type` 😁